libraries/networking/src/DomainHandler.cpp

```diff
@@ -364,17 +366,39 @@ void DomainHandler::setIsConnected(bool isConnected) {
             _lastDomainConnectionError = -1;
             emit connectedToDomain(_domainURL);
 
+            // FIXME: Reinstate the requestDomainSettings() call here in version 2021.2.0 instead of having it in 
```

Comment:
maybe an issue for this?

Reply:
https://github.com/vircadia/vircadia/issues/1099